.editorconfig

```diff
@@ -15,6 +15,14 @@ trim_trailing_whitespace=true
 insert_final_newline=true
 indent_style=space
 indent_size=4
+max_line_length = 100
```

Comment:
I think these changes can all be reverted now that editorconfig on `develop` has been updated.